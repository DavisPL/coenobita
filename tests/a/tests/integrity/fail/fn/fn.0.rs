#![allow(unused_variables)]
#![allow(unused_assignments)]

#[cnbt::tag({*}{*} fn({a}{a}) -> {b}{b})]
fn id(x: i32) -> i32 {
    x
}

fn main() {
    let x = id(5);
}