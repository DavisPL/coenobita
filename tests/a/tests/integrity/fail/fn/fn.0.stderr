error: expected {b}{b}, found {a}{a}
 --> tests/integrity/fail/fn/fn.0.rs:6:5
  |
6 |     x
  |     ^

error: expected {a}{a}, found {root}{root}
  --> tests/integrity/fail/fn/fn.0.rs:10:16
   |
10 |     let x = id(5);
   |                ^
